### DISCUSSION TITLE
What tailwind vscode theme?

### DISCUSSION THREAD

**Discussion by ramiraza** [tailwindlabs/tailwindcss#3630] [Help] [answered]
Does anyone know what theme and font is being used at the image of Tailwind Intellisense extension for vs code? I've included the image below for reference. I really like the rounded corners of the color highlights in the intellisense and the background theme. 

Thanks in advance ☺️

![image](https://user-images.githubusercontent.com/16007576/108643860-3da7f800-7461-11eb-97e7-781f0e8aa5a0.png)

**Comment by Core-Coder** [comment]
I don't know which theme is that, but i have another alternative that is still good. It's called "Git Hub theme", and for the font you can use "Font Switcher" and use the Hack font.

**Comment by smzm** [comment]
[Tokyo night](https://marketplace.visualstudio.com/items?itemName=enkia.tokyo-night) is The theme used in screenshots .

**Reply by cauxNicolas** [reply]
thanks

**Comment by pabloq1** [comment]
Were you able to find it? It's not Tokyo night.

**Reply by Core-Coder** [reply]
I think Tokyo Night match the reference quite perfectly.

**Comment by akkyjumbade** [comment]
It is **Tailwind Moon**
https://marketplace.visualstudio.com/items?itemName=shadowblood.tailwind-moon

**Comment by RvDstudio** [comment]
Hi @ramiraza  i am busy to creating a  vscode theme that looks like the new tailwind docs website [https://tailwindcss.com/](url)
![RhynoTheme](https://user-images.githubusercontent.com/39220977/146087889-4cec5fe4-8b40-4988-9d77-28791f6d7c33.png)

Its still a work in progres  but i am in the good direction.

**Reply by haoolii** [reply]
Nice, It's so exciting

**Reply by aditrachman** [reply]
what theme and font do you use?

**Reply by matagaralph** [reply]
> Hi @ramiraza i am busy to creating a vscode theme that looks like the new tailwind docs website [https://tailwindcss.com/](url) ![RhynoTheme](https://user-images.githubusercontent.com/39220977/146087889-4cec5fe4-8b40-4988-9d77-28791f6d7c33.png)
> 
> Its still a work in progres but i am in the good direction.

Any update on this or achievement buddy.

**Comment by Nabwinsaud** [comment]
> Does anyone know what theme and font is being used at the image of Tailwind Intellisense extension for vs code? I've included the image below for reference. I really like the rounded corners of the color highlights in the intellisense and the background theme.
> 
> Thanks in advance ☺️
> 
> ![image](https://user-images.githubusercontent.com/16007576/108643860-3da7f800-7461-11eb-97e7-781f0e8aa5a0.png)

Tailwind theme search in vs code you will get

**Comment by Yiddishe-Kop** [comment]
It is an actual VScode theme, which is used on [play.tailwindcss.com](https://play.tailwindcss.com/):

<img width="678" alt="CleanShot 2022-07-15 at 10 29 04@2x" src="https://user-images.githubusercontent.com/36875437/179174290-eaa3524e-dd8d-4466-bee6-b0519be805fe.png">

The site is open-source, and by digging into the code I found where the theme colors are being defined: [light theme](https://github.com/tailwindlabs/play.tailwindcss.com/blob/bd6285193ba8acf067102e3db27a1504f902e396/src/monaco/index.js#L88-L137) and [dark theme](https://github.com/tailwindlabs/play.tailwindcss.com/blob/bd6285193ba8acf067102e3db27a1504f902e396/src/monaco/index.js#L139-L188).

And editor customizations other than colors (e.g. the autocomplete widget border-radius etc) are done in this [CSS file](https://github.com/tailwindlabs/play.tailwindcss.com/blob/bd6285193ba8acf067102e3db27a1504f902e396/src/css/main.css).

Seems possible to extract that into a standalone VScode theme 🔥.

**Reply by Yiddishe-Kop** [reply]
OK, I've managed to recreate this theme by a combination of vscode settings customizations & custom css [via [this extension](https://marketplace.visualstudio.com/items?itemName=be5invis.vscode-custom-css)].

Screenshot:

<img width="928" alt="CleanShot 2022-07-16 at 23 13 27@2x" src="https://user-images.githubusercontent.com/36875437/179370650-b9c77da3-518e-4754-b317-23e531764b28.png">

**Reply by EnhaoSun** [reply]
Hi bro, Could you share how did you do that? Thanks!

**Reply by rishu541** [reply]
can you share the same

**Reply by bilgekaana123** [reply]
can you share

**Comment by amirehsan** [comment]
Hi everyone!

Check [this](https://www.tailwindtoolbox.com/guides/tailwind-css-website-vs-code-theme) out and follow the steps.

**Comment by ahmedsomaa** [comment]
This is the closest I got to it. The theme is [Tailwind Colour Themes](https://marketplace.visualstudio.com/items?itemName=shadowblood.tailwind-moon) with the following tweaks found [here](https://www.tailwindtoolbox.com/guides/tailwind-css-website-vs-code-theme)

## Tweaks
```json 
{
  "editor.fontFamily": "Fira Code",
  "editor.fontSize": 12,
  "editor.fontLigatures": true,
  "editor.lineHeight": 2,
  "workbench.colorTheme": "Tailwind Moon Blue"

  "editor.tokenColorCustomizations": {
    "[Tailwind Moon Blue]": {
      "strings": "#7dd3fc",
      "textMateRules": [
        {
          "name": "TW HTML tags",
          "scope": ["punctuation.definition.tag", "punctuation.definition.string.begin.html", "punctuation.definition.string.end.html", "punctuation.separator.key-value.html"],
          "settings": {
            "foreground": "#63738A"
          }
        },
        {
          "name": "TW CSS unit / vue",
          "scope": ["keyword.other", "invalid.illegal.unrecognized-tag.html"],
          "settings": {
            "foreground": "#99F6E4"
          }
        },
        {
          "name": "TW CSS number",
          "scope": " constant.numeric.css",
          "settings": {
            "foreground": "#f8fafc"
          }
        },
        {
          "name": "TW CSS property",
          "scope": " support.type.property-name.css",
          "settings": {
            "foreground": "#7dd3fc"
          }
        },
        {
          "name": "TW Entity",
          "scope": ["entity.name.tag.template.html.vue","entity.name.tag.html"],
          "settings": {
            "foreground": "#F472B6"
          }
        },
        {
          "name": "TW Vue Attributes",
          "scope": "entity.other.attribute-name.html.vue",
          "settings": {
            "foreground": "#A78BFA"
          }
        },
        {
          "name": "TW Entity HTML",
          "scope": "entity.other.attribute-name.html",
          "settings": {
            "foreground": "#CBD5E1"
          }
        }
      ]
    }
  }
}

```

## Demo
![code](https://github.com/user-attachments/assets/b014db80-0f83-417e-ad5d-13acbc138b8c)

**Reply by lannodev** [reply, marked as answer]
## Tailwind Weekly #189: New Catalyst Update, Course Template Sneak Peek, and New Editor Themes coming soon 🔥

### New Tailwind CSS editor themes just around the corner 💻️ 

If you have seen any of the recent Tailwind CSS channel videos, then you probably saw that they have their own custom editor theme. Well, they decided to [share](https://x.com/adamwathan/status/1910800038667968786?utm_source=tailwindweekly.com&utm_medium=newsletter&utm_campaign=tailwind-weekly-189-new-catalyst-update-course-template-sneak-peek-and-new-editor-themes-coming-soon&_bhlid=a144c8f4043c015409c739dea496d5a15cf1f39f) it with everyone and are planning to release it soon 😄
